TG-18

evelynda1985 opened the pull request:
Pense que no tenia que volverlo a crear, aqui ya se corrigio el nombre.

Commit:
TG-13  TG-17
Se corrigio el camelCase en el nombre del metodo calcularValorReceta

Commit:
TG-13  TG-17
Se corrigio el camelCase en el nombre del metodo calcularValorReceta

Commit:
TG-13  TG-18
Se anadieron validadores y optimizacion al metodo buscarRecetasPorIngrediente y se probo y funciono :p

jramirezl commented on src/receta/controller/Chef.java:
```diff
@@ -48,18 +49,29 @@ public double valorReceta(Receta[] recetas,String nombreReceta){
+             if (encontroIngrediente){
+                System.out.println("Puedes usar la receta: " + recetas[i].getNombre() + " con el ingrediente: " + ingrediente);
+             }else{
+                 System.out.println("Lo sentimos, no tenemos recetas con el ingrediente: " + ingrediente);
```
Si es ninguna receta debe haber otro booleano e imprimir esto afuera de los dos for

Commit:
TG-13  TG-19
Se avanzo en el metodo numeroVerduras, se tiene en este momento un error, y es que esta mostrando el mismo resultado en todas todas las cocinas.

se avanzo en el metodo CalcularReceta, hay un error de parsing...

evelynda1985 commented:
Listo, se hizo una validacion extra.
